operators/src/upgrade/k8s/crd/v0.rs

```diff
@@ -162,7 +163,7 @@ impl From<UpgradeState> for String {
 #[derive(Clone, Debug, Default, Serialize, Deserialize, Eq, PartialEq, JsonSchema)]
 pub struct UpgradeActionStatus {
     /// UpgradeAction state.
-    state: Option<UpgradeState>,
+    pub state: UpgradeState,
```

Comment:
You've added a getter below, so presumably don't need this pub now?

Reply:
It needs to be public, or it does not seem to show up, not sure why :thinking:.